clifford/tools/g3c/rotor_parameterisation.py

```diff
@@ -195,8 +195,8 @@ def ga_exp(B):
     Fast implementation of the translation and rotation specific exp function
     """
     if np.sum(np.abs(B.value)) < np.finfo(float).eps:
-        return layout.MultiVector(unit_scalar_mv.value)
-    return layout.MultiVector(val_exp(B.value))
+        return layout.MultiVector(unit_scalar_mv.value, copy=False)
```

Comment:
This might cause problems as the unit_scalar is a constant scoped to the whole file

Reply:
Indeed - however, I don't think this is any worse than the fact that `clifford.g3c.e3` is global, and I could do something careless like
```python
my_vector = e3
my_vector.value[:] = e2.value  # whoops, now e3 == e2
```

Reply:
omg I hate it

Reply:
Yeah I have no strong feeling about layout.blades but the predefined blades definitely have to be readonly

Reply:
```python
eo.value[:] = layout.scalar.value[:]
print("Your CGA is now somewhat PGA")
"""